src/cr/cube/measures/pairwise_pvalues.py

```diff
@@ -18,61 +18,67 @@
 
 
 # pylint: disable=too-few-public-methods
-class PairwisePvalues:
+class PairwiseSignificance:
     """Value object providing matrix of pairwise-comparison P-values"""
 
     def __init__(self, slice_, axis=0, weighted=True):
         self._slice = slice_
         self._axis = axis
         self._weighted = weighted
 
-    @lazyproperty
-    def values(self):
-        """Square matrix of pairwise Chi-square along axis, as numpy.ndarray."""
+    @classmethod
+    def pvals(cls, slice_, axis=0, weighted=True):
+        return cls._factory(slice_, axis, weighted)._pvals
+
+    @staticmethod
+    def _calculate_chi_squared(numel, proportions, margin, observed):
```

Comment:
I was thinking that using the verb in imperative form would be better, since this is the method and not the property. But yeah, I can fix the naming, I agree that `calculate` is not very informative...